src/btree/bt_debug.c

```diff
@@ -881,6 +884,10 @@ __debug_page_metadata(WT_DBG *ds, WT_REF *ref)
     if (mod != NULL)
         WT_RET(ds->f(ds, ", page-state=%" PRIu32, mod->page_state));
     WT_RET(ds->f(ds, ", memory-size %" WT_SIZET_FMT, page->memory_footprint));
+    if (addr != NULL)
+        WT_RET(ds->f(ds, ", start stop ts/txn %s,%s",
+          __wt_time_pair_to_string(addr->oldest_start_ts, addr->oldest_start_txn, tp_string[0]),
+          __wt_time_pair_to_string(addr->newest_stop_ts, addr->newest_stop_txn, tp_string[1])));
```

Comment:
Can't we get here with `WT_REF.addr` pointing onto the page's disk image?

I'm wondering if we should extend `__wt_ref_info()` to return the timestamp/txn information.

Reply:
done in https://github.com/wiredtiger/wiredtiger/pull/5138

Reply:
@luke-pearson, this change may be relevant to you.